src/views/product-landing/components/themed-image/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { ThemedImageProps } from './types'
+
+/**
+ * This component can be used in views where we need to
+ * conditionally render an image based on the theme state.
+ *
+ * See styles/themes/global.css for the attribute handler
+ * that uses `display:none` to conditionally render content
+ * based on the theme state with css
+ */
+export default function ThemedImage({ src, alt }: ThemedImageProps) {
```

Comment:
[See this comment](https://github.com/hashicorp/dev-portal/pull/1752#discussion_r1140836074), I decided to keep this component local for simplicity on this pass and also I don't know enough about the generic needs of this component yet to lift it up globally. Also this is the only known usecase at the moment.